jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -254,6 +257,26 @@ public String getVersion() {
     return project.getVersion().toString();
   }
 
+  void validateBaseImageVersion(@Nullable String baseImage) {
```

Comment:
nit: Naming is a little misleading since we're not explicitly validating all base image version. Would prefer something that matches the meaning of "validate base image version if build uses default base image"